sql/1.110/rollback/rollback.sql

```diff
@@ -0,0 +1,5 @@
+BEGIN;
+
+ALTER TABLE users RENAME COLUMN github_user_plaintext TO github_user;
```

Comment:
Also need to drop the constraint on github_user_fk_github_id, otherwise it won't be possible to drop github_id,  and add again the one on github_user_fk_users_id.